orchestra/proc-macro/src/impl_builder.rs

```diff
@@ -635,9 +635,10 @@ pub(crate) fn impl_feature_gated_items(
 				#(
 					let (#channel_name_tx, #channel_name_rx)
 					=
-						#support_crate ::metered::channel::<
+						#support_crate ::metered::channel_with_priority::<
```

Comment:
Nit: Should we make this configurable per-subsystem ?

Reply:
We can, but I think we'll have little to gain considering the number of subsystems.

Reply:
At least from Polkadot perspective, we only need the priority channels for the networking subsystems, so it makes sense to have it configurable per subsystem.

Reply:
I'd be in favor of having it per subsystem or for everything. We will twice as many channels, in the empty priority channel case, which will be the default for most I presume.